src/author/author.service.ts

```diff
@@ -14,6 +16,7 @@ import { AuthorDto } from './dto/author-update.dto';
 export class AuthorService {
   constructor(
     @InjectRepository(AuthorRepository)
+    @Inject(forwardRef(() => AuthorService))
```

Comment:
Why you use this decorator?

Reply:
I've tried using AuthorModule inside BookModule, but i can't do that